bridges/FilterMoreBridge.php

```diff
@@ -0,0 +1,279 @@
+    const NAME = 'FilterMore';
+    const CACHE_TIMEOUT = 2;
+    const DESCRIPTION = 'Filters a feed of your choice';
+    const URI = 'https://git.lattuta.net/boyska/rss-bridge';
+
+    const PARAMETERS = [
+        [
+        'url' => [
+            'name' => 'Feed URL',
+            'required' => true,
+            'exampleValue' => 'https://example.com/feed/',
+        ],
+        'conj_type' => [
+            'name' => 'Conjunction type type',
+            'type' => 'list',
+            'required' => false,
+            'values' => [
+                'All conditions must be met' => 'and',
+                'Any condition must be met' => 'or',
+            ],
+            'defaultValue' => 'permit',
+        ],
+
+        'title_re' => [
+            'name' => 'Filter item title (regular expression, see php.net/pcre_match for details)',
+            'required' => false,
+            'exampleValue' => '/breaking\ news/i',
+        ],
+        'body_re' => [
+            'name' => 'Filter body (regular expression)',
+            'required' => false,
+            'exampleValue' => '/(technology|politics)/i',
+        ],
+        'author_re' => [
+            'name' => 'Filter author (regular expression)',
+            'required' => false,
+            'exampleValue' => '/(technology|politics)/i',
+        ],
+        'newer_than' => [
+            'name' => 'Filter date: ok if newer than the value (see php.net/strtotime for details)',
+            'required' => false,
+            'exampleValue' => '-14 days',
+        ],
+        'older_than' => [
+            'name' => 'Filter date: ok if older than the value (see php.net/strtotime for details)',
+            'required' => false,
+            'exampleValue' => '-1 hour',
+        ],
+
+        'has_media' => [
+            'name' => 'Has at least 1 media inside',
+            'type' => 'checkbox',
+            'required' => false,
+            'defaultValue' => false,
+        ],
+
+        'invert_filter' => [
+            'name' => 'Invert filter result',
+            'type' => 'checkbox',
+            'required' => false,
+            'defaultValue' => false,
+        ],
+
+        'sort_by' => [
+            'name' => 'Sort by',
+            'type' => 'list',
+            'required' => true,
+            'values' => [
+                "Don't sort" => 'none',
+                'Date' => 'timestamp',
+                'Title' => 'title',
+                'Random' => 'random',
+            ],
+            'defaultValue' => 'timestamp',
+        ],
+        'sort_dir' => [
+            'name' => 'Sort direction',
+            'type' => 'list',
+            'required' => true,
+            'values' => [
+                'Ascending' => 'asc',
+                'Descending' => 'desc',
+            ],
+            'defaultValue' => 'asc',
+        ],
+        'limit' => self::LIMIT,
+
+        ]];
+
+    protected function parseItem($newItem)
+    {
+        $item = parent::parseItem($newItem);
```

Comment:
parent call not necessary

Reply:
oh, didn't know that. what do you mean? can you describe me how I can an `$item` object based on what `FeedExpander` implements?

Reply:
$newItem is already parsed

Reply:
ooops, right